_pages/en_US/troubleshooting.txt

```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>Turning off Parental Controls</u></summary>
+
+You can disable the Parental Controls feature by going to System Settings -> Parental Controls and inserting the PIN, then pressing "Clear Settings" and "Delete" to remove it.
+However, if you do not know the PIN and it is stopping you from accessing the device's settings, you will need to disable it. In order to do this, you need to obtain your device's master key (mkey):
+1. Access [this website](https://mkey.eiphax.tech/)
+1. Fill the following spaces with the information:
++ Device Type: The console you are currently using
```

Comment:
will be 3ds in all cases, being vague will lead to "i have a 2ds and it didn't listed do i select dsi"